### DISCUSSION TITLE
Cross-Section analysis for L-shaped configuration

### DISCUSSION THREAD

**Discussion by Avi2998** [wenbinyugroup/.github#15] [VABS Related Discussions] [answered]
Hi Team, 
I am trying to analyse a L-shaped Composite CS using VABS. The dimensions of the CS along with the geometric and material definition are as follows. 
![image](https://github.com/user-attachments/assets/e38738ed-5222-4a17-873a-7f7a289925fb)![image](https://github.com/user-attachments/assets/6cd07ba7-4d70-4d82-9532-999146d4b33b)

The crosssection, material and baseline .xml files are also attached for reference.

When i run the problem, the Gmsh file opens without any output and i get the following error message in command line. Is there any way to debug this error.
![image](https://github.com/user-attachments/assets/bf4aed9f-010f-4366-81fc-58258e3208d1)

Thanks for the help.
[L Shaped CS.zip](https://github.com/user-attachments/files/17118693/L.Shaped.CS.zip)

Regards,
Avinash

**Comment by unitasium** [comment, marked as answer]
You only need to use one side of the geometry as the "baseline". For example, p6->p7->p8->p1. The thickness of the part (p1-p2) is controlled by the plies of the layup.

In addition, when defining the "arc", it should be "center" instead of "centre".

**Comment by Avi2998** [comment]
Thanks for the answer. I am able to run the file after these modifications.
